types/expect-puppeteer/index.d.ts

```diff
@@ -67,4 +67,11 @@ declare global {
 }
 
 declare function expectPuppeteer(instance: ElementHandle | Page): ExpectPuppeteer;
-export = expectPuppeteer;
+
+interface ExpectDefaultOptions extends ExpectToClickOptions, ExpectTimingActions {}
+declare function defaultOptions(options: ExpectDefaultOptions): ExpectDefaultOptions;
+declare function setDefaultOptions(options: ExpectDefaultOptions): void;
+declare function getDefaultOptions(): ExpectDefaultOptions;
+
+export default expectPuppeteer;
```

Comment:
This package was declared using `module.exports = ...` or some variant. It should be declared with an `export = ...` like one of the following:

### Variables

```ts
declare var xyz: SomeType;

export = xyz;
```

### Classes

```ts
class Foo {
  // ...
}

export = Foo
```

### Functions

```ts
function foo(): any;

export = foo;
```

To import the entity using a default import, you will need to enable `esModuleInterop`. Alternatively, you can just import using a more appropriate syntax like

```ts
import foo = require("foo");
```

[See more here in the FAQ](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9f9f6430117d025aef62ed56128deb65508edc86/README.md#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).